guides/common/modules/ref_registration-methods.adoc

```diff
@@ -0,0 +1,26 @@
+[id="registration-methods_{context}"]
+= Registration Methods
+
+There are the following main methods for registering hosts:
+
+Global registration::
++
+You generate a `curl` command from {Project} and run this command from an unlimited number of hosts to register them using the global registration template.
++
+By using this method, you can also deploy {Project} SSH keys to hosts during registration to {Project} to enable hosts for remote execution jobs.
+For more information about the remote execution jobs, see xref:Configuring_and_Setting_Up_Remote_Jobs_{context}[].
++
+By using this method, you can also configure hosts with Red{nbsp}Hat Insights during registration to {Project}.
+For more information about using Insights with {Project} hosts, see xref:Using_Red_Hat_Insights_with_Hosts_{context}[].
++
+For more information about this method, see xref:Registering_Hosts_{context}[].
+
+*(Deprecated)* Katello CA Consumer::
++
+You download and install the consumer RPM from `_{foreman-example-com}_/pub/katello-ca-consumer-latest.noarch.rpm` and then run Subscription Manager.
```

Comment:
```suggestion
Download and install the consumer RPM from `_{foreman-example-com}_/pub/katello-ca-consumer-latest.noarch.rpm` and then run `subscription-manager`.
```

This only works on EL systems; not Debian/Ubuntu. You run `katello-rhsm-consumer` on Debian/Ubuntu: https://docs.orcharhino.com/or/docs/sources/guides/debian/managing_hosts/registering_hosts.html#Registering_Hosts_Running_Debian_or_Ubuntu_Manually

I'd be happy to contribute after your PR has been merged.

Reply:
I'll try to include it with the methods for non-Satellite.